src/components/Equipment/Scuba/SecondScuba.js

```diff
@@ -128,43 +120,12 @@ function SecondScuba({ classes }) {
           [classes.hidden]: !showDetails
         })}
       >
-        <ClickAwayListener onClickAway={() => setShowDetails(false)}>
-          <div className={classes.content}>
-            <Grid
-              container
-              direction="row"
-              wrap="nowrap"
-              justify="space-between"
-              alignItems="center"
-            >
-              <Typography className={classes.headerTitle}>
-                PART NUMBER 02
-              </Typography>
-              <ButtonBase
-                disableRipple
-                disableTouchRipple
-                onClick={() => setShowDetails(false)}
-              >
-                <img alt="" src={close} className={classes.close} />
-              </ButtonBase>
-            </Grid>
-            <Typography className={classes.title}>Hydrophones</Typography>
-            <Typography className={classes.description}>
-              Waterproof microphones attached to the triangle frame using zip
-              ties.
-            </Typography>
-            <div className={classes.preview}>
-              <img alt="" src={battery} />
-              <ButtonBase
-                disableRipple
-                disableTouchRipple
-                className={classes.search}
-              >
-                <img alt="search" src={search} />
-              </ButtonBase>
-            </div>
-          </div>
-        </ClickAwayListener>
+        <ContentScuba
+          header="PART NUMBER 02"
+          title="Hydrophones"
+          description="Waterproof microphones attached to the triangle frame using zip
+              ties."
+        />
```

Comment:
@nerdlet for click away to work 

```
        <ClickAwayListener onClickAway={() => setShowDetails(false)}>
          <ContentScuba
            header="PART NUMBER 03"
            title="Triangle Frame"
            description="Attached to the waterproof microphones and anchored to the seabed"
          />
        </ClickAwayListener>
```

Remove `ClickAwayListener` from within `ContentScuba`